src/app/components/Curation/CurationGrid/index.tsx

```diff
@@ -36,7 +36,7 @@ const CurationGrid = ({
         </ul>
       ) : (
         <div css={styles.item}>
-          <CurationPromo {...firstPromo} />
+          <CurationPromo {...firstPromo} headingLevel={headingLevel} />
```

Comment:
This has been introduced to fix the integration tests - rather than working around this issue, we need to make sure that the heading levels are correct.